tests/validation/tests/v3_api/test_secrets.py

```diff
@@ -623,12 +623,33 @@ def create_project_client(request):
     namespace["project"] = p
     namespace["c_client"] = c_client
 
+
     def fin():
         client = get_user_client()
         client.delete(namespace["project"])
+
     request.addfinalizer(fin)
 
 
+def rancher_version_check():
```

Comment:
Renaming this function to something like `is_version_greater_than()` would be more explicit and easy to read imo